src/modules/api.js

```diff
@@ -0,0 +1,8 @@
+import displayPhotos from './render.js';
+// -- function to get the information from API
+const getData = async () => {
+  const response = await fetch('https://picsum.photos/v2/list');
+  const data = await response.json();
+  displayPhotos(data);
+};
+getData();
```

Comment:
- I belive that you should export this function in order to reuse this code for another features such as popup comment.
- It would be better if you do not involve displayPhotos function inside this code. Instead of you can return an value and involve getData function inside the displayPhotos, It would be more clear.